sql/core/benchmarks/JsonBenchmark-results.txt

```diff
@@ -3,121 +3,121 @@ Benchmark for performance of JSON parsing
-from_json(timestamp)                                                            4774           4788          13          0.2        4773.6       0.1X
-from_json(date)                                                                 3078           3090          11          0.3        3078.5       0.1X
-infer error timestamps from Dataset[String] with default format                 2025           2058          28          0.5        2025.0       0.1X
-infer error timestamps from Dataset[String] with user-provided format          20261          20338          95          0.0       20260.6       0.0X
-infer error timestamps from Dataset[String] with legacy format                  5495           5528          38          0.2        5495.4       0.0X
+read timestamp text from files                                                   283            289           6          3.5         283.1       1.0X
+read timestamps from files                                                      3364           3431          60          0.3        3363.6       0.1X
+infer timestamps from files                                                     8913           8935          38          0.1        8912.6       0.0X
+read date text from files                                                        263            267           4          3.8         262.9       1.1X
+read date from files                                                            1102           1116          12          0.9        1101.7       0.3X
+timestamp strings                                                                412            426          14          2.4         412.0       0.7X
+parse timestamps from Dataset[String]                                           3941           3956          14          0.3        3940.8       0.1X
+infer timestamps from Dataset[String]                                           9334           9383          43          0.1        9333.8       0.0X
+date strings                                                                     469            484          24          2.1         469.3       0.6X
+parse dates from Dataset[String]                                                1565           1572          11          0.6        1564.8       0.2X
+from_json(timestamp)                                                            5825           5917          88          0.2        5824.5       0.0X
+from_json(date)                                                                 3553           3574          19          0.3        3553.1       0.1X
+infer error timestamps from Dataset[String] with default format                 2590           2609          19          0.4        2589.9       0.1X
+infer error timestamps from Dataset[String] with user-provided format           2517           2551          30          0.4        2516.8       0.1X
```

Comment:
@MaxGekk The bencnmark updated. `infer error timestamps from Dataset[String] with user-provided format` speed already up.